actual.js

```diff
@@ -30,7 +30,12 @@ async function initialize(config) {
             config.set("budget_id", id)
         }
 
-        await actual.downloadBudget(id);
+        if (appConfig.ACTUAL_SERVER_IS_ENCRYPTED) {
+            await actual.downloadBudget(id, { password: appConfig.ACTUAL_SERVER_PASSWORD });
```

Comment:
Is this assuming that the file password is the same as the server password?  Shouldn't there be a new variable for the encryption password?

Reply:
I've updated this.